lib/enkf/model_config.cpp

```diff
@@ -506,6 +508,7 @@ void model_config_init(model_config_type * model_config ,
 
     model_config->obs_config_file = util_alloc_string_copy(obs_config_file);
   }
+  // Only subst_list_free for non const. Free const? (define_list);
```

Comment:
config should be responsible for cleaning up define_list

Reply:
As this is the initiation of the `ModelConfig` it _should_ free it? We are doing a deep copy of the list within `ext_job_set_define_args`, and as such the `ext_job` will not need it.

Reply:
The define_list is owned by the config_content object. I might be passed on to some other config object at a later stage.